### PR TITLE
Create release note for release 3.7.fp5

### PR DESCRIPTION
**Pull request by AliceAlbano**

### CODE REVIEW

File: admin/en/source/pages/changelog.rst [hunk 1]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7361 Windows generation error when disk too small does not raise an understandable error message
+* 7620 db_modifs and postupdate modifs applied to several versions of UForge are not handled properly by update_mechanism
+* 7608 qtbase recipe fails during oe-lite compilation
+* 7758 On the generation UI pages, there is a CSS issue between headers and content
+* 7853 License of WS2008R2 is displayed on WS2012R2
+* 7771 Hover message on items in scan list is not internationalized
+* 7871 Updating the certificate of a google cred account generates a null pointer exception and "this should never happen, please updateTemplateInfo" in the UI
+* 7682 Published image tag summary displayed wrong tooltip
+* 7635 Password input field of artifact accounts
+* 7584 Fix bug Applications and Services for windows are not parsed correctly
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: remove "Fix bug" and Windows

File: admin/en/source/pages/changelog.rst [hunk 2]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7767 Fix missing timezone data on branch master
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: remove "Fix"

File: admin/en/source/pages/changelog.rst [hunk 3]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7897 500 error occurs in image generation using a template without a partition table
+* 7669 Bug in ScanActionHandler of eventcontroller
+* 6285 AWS publication is not working behing an external http proxy
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: should be "behind"

File: admin/en/source/pages/changelog.rst [hunk 4]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7630 Outscale publish connector is not working anymore
+* 7298 Import/Export Software bundle fails with "Permission denied error"
+* 7820 Cli rpm generation on Jenkins is failing
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: is failing should be "fails"

File: admin/en/source/pages/changelog.rst [hunk 5]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 6789 When booting a migrated instance, haldaemon starts although autostart setting is off in the source machine
+* 6331 Remove log messages not "Futjitsu way" compatible
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: not sure we should list this type of bug for external documentation visible by client. This is not a "bug" per say, just an internal improvement that clients should not be aware of

File: admin/en/source/pages/changelog.rst [hunk 6]
```diff
@@ -6,17 +6,39 @@ Changelog
...
 Bug Fixes
 ~~~~~~~~~
-
+* 7560 Oracle Linux is treated as RHEL at scan
+* 7622 OpenSUSE generation Failed with default OS Profile due to package conflict.
+* 7423 CentOS 7.3 VBox image never ends up booting if '/' partition is a logical volume
+* 7429 Error message is always logged in oar error log "unary operator expected" when generating Linux image.
+* 7361 Windows generation error when disk too small does not raise an understandable error message
+* 7620 db_modifs and postupdate modifs applied to several versions of UForge are not handled properly by update_mechanism
+* 7608 qtbase recipe fails during oe-lite compilation
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: please remove the 7608

File: admin/en/source/pages/changelog.rst [hunk 7]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7758 On the generation UI pages, there is a CSS issue between headers and content
+* 7853 License of WS2008R2 is displayed on WS2012R2
+* 7771 Hover message on items in scan list is not internationalized
+* 7871 Updating the certificate of a google cred account generates a null pointer exception and "this should never happen, please updateTemplateInfo" in the UI
+* 7682 Published image tag summary displayed wrong tooltip
+* 7635 Password input field of artifact accounts
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: 7635: The title does not explain what was the problem

File: admin/en/source/pages/changelog.rst [hunk 8]
```diff
@@ -6,17 +6,39 @@ Changelog
...
+* 7584 Fix bug Applications and Services for windows are not parsed correctly
+* 7767 Fix missing timezone data on branch master
+* 7897 500 error occurs in image generation using a template without a partition table
+* 7669 Bug in ScanActionHandler of eventcontroller
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: 7669: I think we can rework this one (and remove Bug at the beginning)

File: admin/en/source/pages/changelog.rst (hunk 4 above)
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: 7820: please remove it, this is internal

File: admin/en/source/pages/changelog.rst (hunk 5 above)
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: agree

**Comment by AliceAlbano** [comment]
Thank you for you reviews, I have committed a new version for the release note with your comments

File: admin/en/source/pages/changelog.rst [hunk 9]
```diff
@@ -22,23 +22,18 @@ Bug Fixes
...
+* 7635 The type of the password input field of artifact accounts is inconsistent in the UI
+* 7584 Applications and Services for Windows are not parsed correctly
+* 7767 Missing timezone data on branch master
 * 7897 500 error occurs in image generation using a template without a partition table
-* 7669 Bug in ScanActionHandler of eventcontroller
-* 6285 AWS publication is not working behing an external http proxy
+* 7669 The EventController service does not consume ScanAction event correctly
+* 6285 AWS publication is not working behind an external http proxy
 * 7630 Outscale publish connector is not working anymore
-* 7298 Import/Export Software bundle fails with "Permission denied error"
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: yo keep

File: admin/en/source/pages/changelog.rst [hunk 10]
```diff
@@ -22,23 +22,18 @@ Bug Fixes
...
-* 7820 Cli rpm generation on Jenkins is failing
-* 6789 When booting a migrated instance, haldaemon starts although autostart setting is off in the source machine
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: to keep

**Pull request merged**